README.md

```diff
@@ -1,13 +1,32 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Makes use of new skills in Express and Mongodb to build an API which includes both GET request endpoints to return data and POST endpoints to create data.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+In order to replace the API we built, you're going to need to build a Thought Mongoose model which has properties for the message string, a heart property for tracking the number of likes, and a createdAt property to store when the thought was added.
+
+Then, you'll need to add 3 endpoints:
+
+In summary I got stuck on two things, MongoDB Atlas, was a new cluster needed or not? And I posted the first data to Mongo with test string instead of project name. Had to drop it but after that it worked fine.
+
+The other issue was that I wrote the post request in slighty different way than provided guidance and got a bit stuck, first with naming and then with a missing curly {} :)
+
```

Comment:
It would be nice to read more about how you planned your project and what tools and techniques you used. Also, it would be fun to know what you would’ve done if you had more time - maybe you wanted to do something more on the frontend? 😊